renderdoc/data/glsl/texdisplay.frag

```diff
@@ -223,7 +223,7 @@ void main(void)
 			if(texdisplay.Channels.a == 1.0f)
 				col = vec4(col.aaa, 1);
 			else
-				col = vec4(vec3(dot(col.rgb, vec3(1.0f))), 1.0f);
+				col = vec4(vec3(dot(col.rgb, texdisplay.Channels.rgb)), 1.0f);
```

Comment:
What's the reasoning behind this change? I think it should be harmless, but I want to understand the intention. this code happens after the `texdisplay.Channels` has already been applied, so here `col` should only have one channel present, and the rest should be `0.0`. See lines 173-176 above.